karma.conf.js

```diff
@@ -26,6 +26,7 @@ module.exports = function (config) {
     logLevel: config.LOG_INFO,
     autoWatch: true,
     browsers: ['ChromeHeadless', 'Chrome'],
+    browserNoActivityTimeout: 30000,
```

Comment:
I think this comes from our trials of `ng test` and where the timeout issue was coming from. It turned out it wasn't from the insufficient time for tests, so I think we can leave this as default here?

That means either removing it, or setting it to `10000` - but it's not something that should block this PR.

Reply:
Ah indeed; let's remove it for now.